ros2node/ros2node/verb/get_type_description.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2023 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import rclpy
+from ros2cli.node import NODE_NAME_PREFIX
+from ros2cli.node.strategy import add_arguments
+from ros2node.api import (
+    NodeNameCompleter,
+    parse_node_name,
+)
+from ros2node.verb import VerbExtension
+from type_description_interfaces.msg import FieldType
+from type_description_interfaces.srv import GetTypeDescription
+
+
+def print_field_type(ft):
```

Comment:
The function name `print_field_type` is a bit misleading since it doesn't print anything.
IMO it would be more appropriate to rename it to the `get_field_type_str` or something similar.